test/runtests.jl

```diff
@@ -42,8 +42,7 @@ const TRIXI_NTHREADS = clamp(Sys.CPU_THREADS, 2, 3)
         # If the process errors out the testset would error out as well,
         # cf. https://github.com/JuliaParallel/MPI.jl/pull/391
         @test true
-
-        run(`$(Base.julia_cmd()) --threads=$TRIXI_NTHREADS --check-bounds=yes --code-coverage=none $(abspath("test_threaded.jl"))`)
+        run(`$(Base.julia_cmd()) --threads=$TRIXI_NTHREADS --check-bounds=yes $(abspath("test_threaded.jl"))`)
```

Comment:
Why is this changed in this PR? This seems unrelated to the original purpose of the PR?!

Reply:
It was a test discussed in trixi-students to enable coverage for threaded CI and MPI on Linux. It is seems to work without issues, I'll publish new PR for that.

Reply:
OK, thanks!